libs/api/domains/air-discount-scheme/src/dto/getDiscount.input.ts

```diff
@@ -0,0 +1,9 @@
+import { Field, InputType } from '@nestjs/graphql'
+import { IsString } from 'class-validator'
+
+@InputType()
+export class GetDiscountInput {
+  @IsString()
+  @Field()
+  nationalId!: string
+}
```

Comment:
This file isn't being used I think. Will be deleted.

Reply:
Yup, not needed since the CurrentUser decorator contains the relevant national id